flink-test-utils-parent/flink-test-utils/src/main/java/org/apache/flink/test/util/MiniClusterPipelineExecutorServiceLoader.java

```diff
@@ -61,8 +61,7 @@ public MiniClusterPipelineExecutorServiceLoader(MiniCluster miniCluster) {
      * MiniClusterPipelineExecutorServiceLoader}.
      */
     public static Configuration createConfiguration(
-            Collection<Path> jarFiles, Collection<URL> classPaths) {
-        Configuration config = new Configuration();
+            Configuration config, Collection<Path> jarFiles, Collection<URL> classPaths) {
```

Comment:
Do we want to add some log statements in case we overwrite some properties set provided by the user?

Reply:
Thanks, that is a good idea, I'll add that.

Reply:
Maybe we could also rename the method to `updateConfiguration`/`prepareConfigurationForMiniClusterExecution` or something better because now the method no longer creates a configuration.